style.css

```diff
@@ -0,0 +1,4 @@
+body {
+    background: rgb(238, 174, 202);
+    background: radial-gradient(circle, rgba(238, 174, 202, 0.774969362745098) 0%, rgba(148, 187, 233, 0.5340730042016807) 100%);
```

Comment:
maybe we could simplify with:
background: radial-gradient(circle, rgba(238, 174, 202, 0.8) 0%, rgba(148, 187, 233, 0.5) 100%);
?

Reply:
Isn't risky to not specify a basic background color (rgb(238, 174, 202)) in case some browsers couldn't properly render or read the gradient pattern?